art/estimators/encoding/tensorflow1.py

```diff
@@ -0,0 +1,151 @@
+# MIT License
```

Comment:
Please rename this file to `tensorflow.py` to make it consistent with other files for TensorFlow 1.x